vms/proposervm/state/block_state.go

```diff
@@ -109,7 +119,7 @@ func (s *blockState) GetBlock(blkID ids.ID) (block.Block, choices.Status, error)
 	}
 
 	// The key was in the database
-	blk, err := block.Parse(blkWrapper.Block)
+	blk, err := block.Parse(blkWrapper.Block, s.durangoTime)
```

Comment:
We could avoid passing the timestamp around here if we wanted to just provide the zero value. There is no reason for us to perform additional verification against certificates that have already been accepted on chain.